### PR TITLE
Fix variant quantity that's put in the cart

### PR DESCRIPTION
**Pull request by StefanSlehta**
Clicking `add to cart` with any quantity was always adding 1 to the cart.

### CODE REVIEW

File: test/unit/product.js [hunk 1]
```diff
@@ -749,4 +766,35 @@ describe('Product class', () => {
...
+      return product.init(testProductCopy).then((product) => {
+        product.cart.model.lineItems = [];
+        product.cart.props.client = product.props.client;
+        return Promise.resolve();
+      });
+    });
+
+    it('add variant to cart is called with the right quantity of selected variant', () => {
+      product.selectedQuantity = 1111;
+      const crt = sinon.stub(product.cart, 'addVariantToCart');
```
Comment (minasmart) [inline test/unit/product.js]: full words please

File: test/unit/product.js [hunk 2]
```diff
@@ -749,4 +766,35 @@ describe('Product class', () => {
...
+
+      let evt = new Event('click shopify-buy__btn--parent');
```
Comment (minasmart) [inline test/unit/product.js]: this is never reassigned. why is it a `let`?

**Comment by SpankyWorks** [comment]
This is a duplicate of #455

**Comment by minasmart** [comment]
@spankyworks #455 has no tests. This branch is tested.

**Pull request merged**